index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+    <div>
+        <img src="" alt="">
+        <p>Front-End Tools</p>
+    </div> 
+
+</body>
+</html>
```

Comment:
add an empty line at the end of file, this can be configured in your text editor to be added automatically.